Public/Src/Cache/ContentStore/Distributed/NuCache/LocalLocationStore.cs

```diff
@@ -1055,11 +1055,11 @@ public async Task<BoolResult> TrimBulkAsync(OperationContext context, IReadOnlyL
                 () =>
                 {
                     var effectiveLastAccessTimes = new List<ContentHashWithLastAccessTimeAndReplicaCount>();
-                    int replicaCount = 1;
 
                     foreach (var contentHash in contentHashes)
                     {
                         DateTime? effectiveLastAccessTime = null;
+                        int replicaCount = 1;

```

Comment:
Just curious, does it make sense to use nullable int and actually use null if we don't know for sure the number of replicas?

Reply:
Decided not to make nullable since it would add no value and we would have to add null checks in more places.